scripts/release/utils.js

```diff
@@ -115,6 +125,35 @@ const getChecksumForCurrentRevision = async cwd => {
   return hashedPackages.hash.slice(0, 7);
 };
 
+const getCommitFromCurrentBuild = async () => {
+  const cwd = join(__dirname, '..', '..');
+
+  // If this build includes a build-info.json file, extract the commit from it.
+  // Otherwise fall back to parsing from the package version number.
+  // This is important to make the build reproducible (e.g. by Mozilla reviewers).
+  const buildInfoJSON = join(
+    cwd,
```

Comment:
'Once 4783999696835b31ad9f83b93e7cc1c8edd77064 lands you can alternatively read from `build2/COMMIT_SHA`

Reply:
Oh nice. Fair.